src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FtpaDecisionMidEvent.java

```diff
@@ -47,7 +49,7 @@ public boolean canHandle(
             .orElseThrow(() -> new IllegalStateException("FtpaApplicantType is not present"));
 
         Optional<String> ftpaSubmitted = asylumCase.read(
-            ftpaApplicantType.equals("appellant") == true ? FTPA_APPELLANT_SUBMITTED : FTPA_RESPONDENT_SUBMITTED, String.class);
+            ftpaApplicantType.equals("appellant") ? FTPA_APPELLANT_SUBMITTED : FTPA_RESPONDENT_SUBMITTED, String.class);
 
```

Comment:
"appellant" - Shouldn't this be a constant ?

Reply:
It is again defined as String type and there is no constant defined.

Reply:
Sorry - can't it not be Parties.APPELLANT ?